docs/guides/linux_thread_connectivity.md

```diff
@@ -0,0 +1,147 @@
+# Configuring Thread network on a Linux machine
+
+In order to use a CHIP controller such as
+[Python CHIP Controller](../../src/controller/python/README.md) to control
+CHIP-enabled Thread devices from a Linux host, the host must be equipped with a
+Thread radio device, a new Thread network must be created and packets addressed
+to devices within the network must be routed via the radio device. The article
+describes how to achieve that on a Linux machine with Ubuntu 20.04, using
```

Comment:
I assume the same steps will work on Ubuntu 20.10?

The reason it's significant is we had to bump our requirement from 20.04 to 20.10 to get a usable bluez stack. So if you want to use the same distro for both you'll want 20.10.

Reply:
Right, it works equally well on 20.10. Changed to "Ubuntu 20.04 or later".